src/Pages/SettingsPage/SettingsPage.js

```diff
@@ -0,0 +1,40 @@
+import LogoutButton from "Components/LogoutButton/LogoutButton";
+import SpotifyLoginButton from "Components/SpotifyLoginButton/SpotifyLoginButton";
+import { Navigate } from "react-router";
+import useAuthentication from "Services/firebase/useAuthentication";
+import useSpotifyAuthData from "Store/selectors/useSpotifyAuthData";
+import useTitle from "Utils/useTitle";
+
+const SettingsPage = function () {
+  useTitle("Bragi 3000 - Settings");
+
+  const { authReady, loggedIn, user } = useAuthentication();
+
+  const spotifyAuthData = useSpotifyAuthData();
+
+  if (!authReady)
+    return (
+      <span>Waiting for auth...</span>
+    );
+
+  if (!loggedIn)
+    return (
+      <Navigate replace to="/login" />
+    );
+
+  return (
+    <div>
+      <span>Your token: {user.uid}</span>
```

Comment:
This token is not changing over time right?

Reply:
I should rephrase my question, is the token on which the firebase Auth is based on the same as the spotify one? As it is only valid for a certain time period right?

Reply:
Yeah, token is not the right term here. This uid is unique for the (firebase) user and stays the same forever. The changing spotify token is stored in firebase using this uid.

Reply:
Firebase also uses tokens for its own authentication, but that's managed by the packages under the hood, so no need to take care of those.